test/integration/targets/tower_workflow_template/tasks/main.yml

```diff
@@ -92,14 +164,25 @@
     that:
       - "result is changed"
 
-- name: Check module fails with correct msg
+- name: Check module fails with correct msg (organization)
   tower_workflow_template:
     name: my-workflow
     organization: Non Existing Organization
-    schema: '[{"success": [{"job_template": "my-job-1"}], "job_template": "my-job-2"}]'
+    state: present
+  register: result
+  ignore_errors: true
+
+- assert:
+    that:
+      - "result.msg =='Failed to update workflow job template: The requested object could not be found.'"
+
+- name: Check module fails with correct msg (inventory)
+  tower_workflow_template:
+    name: my-workflow
+    inventory: Non Existing Inventory
   register: result
   ignore_errors: true
 
 - assert:
     that:
-      - "result.msg =='Failed to update organization source,organization not found: The requested object could not be found.'"
\ No newline at end of file
+      - "result.msg =='Failed to update workflow job template: The requested object could not be found.'"
```

Comment:
Can you add tests for `ask_inventory` and `ask_extra_vars` as well?

Reply:
Thanks a lot for reviewing! I added integration tests for `ask_inventory` and `ask_extra_vars` parameters.